docs/pages/kubernetes-access/controls.mdx

```diff
@@ -220,17 +220,81 @@ spec:
     - viewers
     kubernetes_users:
     - myuser
+    - system:serviceaccount:someNamespace:saName # Service account name
     # ...
   deny: {}
 ```
 
 The value of `kubernetes_groups` and `kubernetes_users` is a list of names of
-groups and users to enable impersonation for.
+groups, users or service accounts to enable impersonation for.
 
-#### How Teleport users impersonate Kubernetes users and groups
+#### Kubernetes Users and Groups
 
-There are two ways for an end user to specify which user and groups to
-impersonate:
+Kubernetes Users and Groups are entities that exist in
+the Kubernetes Cluster and for which permissions are controlled through
+`ClusterRoleBinding` or `RoleBinding` resources. If by any chance they do not
+exist in the cluster, Kubernetes RBAC will ignore them.
+
+An example of a `ClusterRoleBinding` resource that assigns the built-in `view`
+`ClusterRole` to a Group `cluster-viewer-group` and to a User `cluster-viewer-user`
+can be found below.
```

Comment:
For a more natural voice:

```suggestion
Here's an example of a `ClusterRoleBinding` resource that assigns the built-in `view`
`ClusterRole` to a Group `cluster-viewer-group` and to a User `cluster-viewer-user`:
```

Reply:
P.S. Just want to confirm that `User` and `Group` in this context are API objects, warranting the casing used here per the [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).